app/views/case_workers/admin/management_information/_stats_report.html.haml

```diff
@@ -1,4 +1,5 @@
 - i18n_scope = 'case_workers.admin.management_information.index'
+- action_context = t("report_types.#{report_object}_html", scope: i18n_scope)
```

Comment:
i was trying to avoid any dynamic translation lookup as it is not testable by i18n-tasks gem. will need to look at this to see if we can break it down to avoid that